chromium_src/chrome/browser/history/history_utils.cc

```diff
@@ -5,11 +5,16 @@
 
 #define CanAddURLToHistory CanAddURLToHistory_ChromiumImpl
 #include "../../../../../chrome/browser/history/history_utils.cc"  // NOLINT
+#include "brave/common/extensions/extension_constants.h"
```

Comment:
nit: We don't need to add this header including between our `def` and `undef`.
`extension_constants.h` is not directly related with `CanAddURLToHistory` redefinition.
It would be better to move to below copyright section.

Reply:
I think this include file could only be used when extensions are enabled, this is why the android build is broken in this PR as below because extensions are not enabled in Android builds, you probably need a guard for accessing ethereum_remote_client_extension_id and this header include.

```
19:01:48  ld.lld: error: undefined symbol: ethereum_remote_client_extension_id
19:01:48  >>> referenced by history_utils.cc:0 (/home/ubuntu/jenkins/workspace/build-pr_exclude-wallets-history/src/brave/chromium_src/chrome/browser/history/history_utils.cc:0)
19:01:48  >>>               thinlto-cache/Thin-6bcd54.tmp.o:(CanAddURLToHistory(GURL const&))
```

Reply:
include removed from the `CanAddURLToHistory` definition, and guards have been added for Android. Thanks!

Reply:
I think `#if BUILDFLAG(ENABLE_EXTNSION)` is more proper than os guard.

Reply:
done